package.json

```diff
@@ -103,7 +103,8 @@
     "**/semver": "^7.5.3",
     "**/set-value": "^4.1.0",
     "**/xml2js": "^0.5.0",
-    "**/yaml": "^2.2.2"
+    "**/yaml": "^2.2.2",
+    "**/@babel/traverse": "^7.23.2"
```

Comment:
It doesn't seem to me like a resolution is necessary. All the versions in the lock file are compatible with this version, so it should only need to be an update to the lock file.

With a grain of salt, though. I don't know how this project handles CVE's in terms of what optimizer does/guaranteeing that all plugins use this version too

Reply:
I think adding package resolution will install the dependencies rightly to the desired version. Also, I think its a [standard practice ](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/SECURITY.md)to add package resolution and to avoid directly editing the lock file as it can break somethings. I don't know how this package is any different from other ones we have dealt with.

Reply:
Yeah, @BSFishy is right that it's not necessary; simply removing the entries from the `yarn.lock` and re-running the bootstrap command would be sufficient. But for CVEs, I think it's kind of nice to enforce a minimum range via a resolution, in case some dep decided to go backwards, so I'm fine with it either way.